src/components/Give/index.js

```diff
@@ -149,20 +148,24 @@ class App extends Component {
           heritageContractAddress
         );
 
-        heritage.methods
-          .makeDonation(donationId)
-          .send({
-            from: accounts[0],
-            value: web3.utils.toWei(donationAmount, 'ether')
-          })
-          .then(response => {
-            console.log('Metamask response' + response.toString());
-            this.sendDonaterInfo();
-          })
-          .catch(err => {
-            this.setState({ errorMessage: noWeb3ErrorMessage });
-            // console.log(err);
-          });
+        heritage.methods.getDonation(animalId).call().then((token) => {
+          donationId = token._originalDonationId;
```

Comment:
@jschiarizzi hope this is what you wanted!

Reply:
No this should be 
fundraiserID = token._originalDonationId;